.buildkite/test-e2e.yml

```
@@ -1,4 +1,4 @@
-- label: 'Test E2E (nightly operator)'
+- label: 'Test E2E (nightly operator) - RayCluster & GCS'
```

```suggestion
- label: 'Test RayCluster and GCS E2E (nightly operator)'
```

nit: align naming format with others